src/ensembl/src/content/app/browser/drawer/Drawer.tsx

```diff
@@ -32,12 +39,23 @@ type OwnProps = {};
 type DrawerProps = StateProps & DispatchProps & OwnProps;
 
 const Drawer: FunctionComponent<DrawerProps> = (props: DrawerProps) => {
+  const getTrackDetails = () => {
+    const childTracks = props.ensObjectTracks['child_tracks'];
+
+    return find(childTracks, { track_id: props.drawerView });
+  };
+
   const getDrawerViewComponent = () => {
     switch (props.drawerView) {
-      case 'gene':
+      case 'gene-feat':
         return <DrawerGene ensObjectInfo={props.ensObjectInfo} />;
-      case 'transcript':
-        return <DrawerTranscript ensObjectInfo={props.ensObjectInfo} />;
+      case 'gene-feat-1':
```

Comment:
wow, where did these names come from? Is whoever sending them going to change them?

Reply:
I don't think this will change until we support multiple transcripts. At the moment it all seems to be 'gene-feat-1'.